### DISCUSSION TITLE
How Do I Disable The Blurred Edges On The Pen Tool?

### DISCUSSION THREAD

**Discussion by Murnjendoof** [bitbof/klecks#73] [Q&A] [answered]
I would like to disable the blurred edges on the pen tool, and just draw with one solid colour. I have tried using the "lock alpha" button but I can't tell what it does, as it doesn't seem to affect the blurred edges on the brush. I have also tried using the Pixel tool, but it can be quite limiting as it only comes in a square shape, and I would very much like to use a circular brush.

**Comment by bitbof** [comment, marked as answer]
Not possible atm. Pixel brush will get a circle shape when possible. It was easiest to implement the square shape.

**Reply by Murnjendoof** [reply]
Great to hear, thanks for the response. Hope to see a circular pixel brush soon, and thanks for all of your hard work on Kleki!
